browser/src/control/Control.NotebookbarImpress.js

```diff
@@ -428,6 +428,34 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 				'text': _UNO('.uno:Sidebar'),
 				'command': '.uno:SidebarDeck.PropertyDeck'
 			},
+			{
+				'type': 'container',
+				'children': [
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'showslide',
+								'type': 'menubartoolitem',
+								'text': _UNO('.uno:ShowSlide', 'presentation'),
+								'command': '.uno:ShowSlide'
+							}
+						]
+					},
+					{
+						'type': 'toolbox',
+						'children': [
+							{
+								'id': 'hideslide',
+								'type': 'menubartoolitem',
+								'text': _UNO('.uno:HideSlide', 'presentation'),
+								'command': '.uno:HideSlide'
+							}
+						]
+					}
+				],
+				'vertical': 'true'
+			}
```

Comment:
Could we please use `'type': 'bigtoolitem'` instead for each one and show just one each time (depending on if isHiddenSlide)